core/services/chainlink/application.go

```diff
@@ -80,6 +83,8 @@ import (
 	"github.com/smartcontractkit/chainlink/v2/plugins"
 )
 
+const ApplicationHeartbeatSeconds = 1
```

Comment:
Shouldn't this already by `1 * time.Second`? At least based on the naming?

Reply:
durations can not be initialized as a `const`, which I'd like to preserve. I'm open to other names. what would you suggest?

Reply:
What do you mean? This is valid:
```suggestion
const ApplicationHeartbeat = time.Second
```
It is no longer an _untyped_ `const`, but does that matter?

Reply:
hmm. i was fairly certain the last time I tried to do something similar in the workflow engine I was unable to use a const to capture something like (5 * time.Second).